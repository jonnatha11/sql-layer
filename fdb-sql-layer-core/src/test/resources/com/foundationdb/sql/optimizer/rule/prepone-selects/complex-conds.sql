SELECT *
 FROM customers INNER JOIN orders ON customers.cid = orders.cid
 WHERE UPPER(name) = 'SMITH'
   AND (DATE(order_date) = '2011-01-01' OR DATE(order_date) = '2012-01-01')