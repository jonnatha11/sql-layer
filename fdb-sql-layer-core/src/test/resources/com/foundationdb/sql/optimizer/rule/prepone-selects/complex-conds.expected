SelectQuery@29a3139
  ResultSet@59159684[cid, name, oid, cid, order_date, special]
    Project@184ce1cd[customers.cid, customers.name, orders.oid, orders.cid, orders.order_date, orders.special]
      Select@5f7c6631[]
        Flatten@3b18618d(customers INNER orders)
          Select@3dac8e4a[upper(customers.name) == SMITH, or(date(orders.order_date) == 2011-01-01,date(orders.order_date) == 2012-01-01)]
            GroupScan@bb1bbe8(Group(test.customers), rows = 1000, cost = 61641.0)
              TableSource@114662e3(customers - TableGroup@41a78257(customers))
              TableSource@24062df1(orders - TableGroupJoin@5c5e3f6d(Join(test/customers/cid/test/test.orders/cid: test.orders -> test.customers, group(test.customers))))